vendor.mod

```diff
@@ -113,8 +113,9 @@ require (
 	github.com/beorn7/perks v1.0.1 // indirect
 	github.com/cenkalti/backoff/v4 v4.1.2 // indirect
 	github.com/cespare/xxhash/v2 v2.1.2 // indirect
-	github.com/cilium/ebpf v0.7.0 // indirect
+	github.com/cilium/ebpf v0.9.1 // indirect
 	github.com/container-storage-interface/spec v1.5.0 // indirect
+	github.com/containerd/cgroups v1.0.4 // indirect
```

Comment:
I guess most of these will go away if we update containerd vendor to 1.7 (I need to update my PR https://github.com/moby/moby/pull/44530).

Was giving the hcsshim one a try (https://github.com/microsoft/hcsshim/pull/1618), but didn't get too far.

```
go mod graph --modfile=vendor.mod | grep ' github.com/containerd/cgroups'

github.com/docker/docker github.com/containerd/cgroups@v1.0.4
github.com/docker/docker github.com/containerd/cgroups/v3@v3.0.1
github.com/Microsoft/hcsshim@v0.9.7 github.com/containerd/cgroups@v1.0.1
github.com/containerd/containerd@v1.6.19 github.com/containerd/cgroups@v1.0.4
github.com/containerd/nydus-snapshotter@v0.3.1 github.com/containerd/cgroups@v1.0.3
github.com/moby/buildkit@v0.11.4 github.com/containerd/cgroups@v1.0.4
github.com/containerd/containerd@v1.5.7 github.com/containerd/cgroups@v1.0.1
github.com/Microsoft/hcsshim@v0.8.21 github.com/containerd/cgroups@v1.0.1
github.com/containerd/containerd@v1.5.0-beta.3 github.com/containerd/cgroups@v0.0.0-20210114181951-8a68de567b68
github.com/Microsoft/hcsshim@v0.8.10 github.com/containerd/cgroups@v0.0.0-20200531161412-0dbf7f05ba59
github.com/containerd/stargz-snapshotter@v0.0.0-20201027054423-3a04e4c2c116 github.com/containerd/cgroups@v0.0.0-20200710171044-318312a37340
github.com/containerd/containerd@v1.5.1 github.com/containerd/cgroups@v1.0.1
github.com/Microsoft/hcsshim@v0.8.16 github.com/containerd/cgroups@v0.0.0-20210114181951-8a68de567b68
github.com/containerd/containerd@v1.5.0-rc.0 github.com/containerd/cgroups@v0.0.0-20210114181951-8a68de567b68
github.com/Microsoft/hcsshim@v0.8.15 github.com/containerd/cgroups@v0.0.0-20200824123100-0b889c03f102
github.com/containerd/nri@v0.0.0-20201007170849-eb1350a75164 github.com/containerd/cgroups@v0.0.0-20200710171044-318312a37340
github.com/Microsoft/hcsshim@v0.8.9 github.com/containerd/cgroups@v0.0.0-20190919134610-bf292b21730f
github.com/containerd/containerd@v1.5.0-beta.4 github.com/containerd/cgroups@v0.0.0-20210114181951-8a68de567b68
github.com/containerd/containerd@v1.5.0-beta.1 github.com/containerd/cgroups@v0.0.0-20200824123100-0b889c03f102
github.com/Microsoft/hcsshim@v0.8.14 github.com/containerd/cgroups@v0.0.0-20200531161412-0dbf7f05ba59
github.com/Microsoft/hcsshim@v0.8.7 github.com/containerd/cgroups@v0.0.0-20190919134610-bf292b21730f
github.com/containerd/imgcrypt@v1.0.1 github.com/containerd/cgroups@v0.0.0-20190717030353-c4b9ac5c7601
```

Reply:
With containerd 1.7.0-rc.2, it looks like;

```
go mod graph --modfile=vendor.mod | grep ' github.com/containerd/cgroups@'
github.com/docker/docker github.com/containerd/cgroups@v1.1.0
github.com/Microsoft/hcsshim@v0.10.0-rc.7 github.com/containerd/cgroups@v1.1.0
github.com/containerd/containerd@v1.7.0-rc.2 github.com/containerd/cgroups@v1.1.0
github.com/containerd/nydus-snapshotter@v0.3.1 github.com/containerd/cgroups@v1.0.3
github.com/moby/buildkit@v0.11.4 github.com/containerd/cgroups@v1.0.4
github.com/Microsoft/hcsshim@v0.8.10 github.com/containerd/cgroups@v0.0.0-20200531161412-0dbf7f05ba59
github.com/containerd/stargz-snapshotter@v0.0.0-20201027054423-3a04e4c2c116 github.com/containerd/cgroups@v0.0.0-20200710171044-318312a37340
github.com/Microsoft/hcsshim@v0.8.9 github.com/containerd/cgroups@v0.0.0-20190919134610-bf292b21730f
github.com/Microsoft/hcsshim@v0.8.7 github.com/containerd/cgroups@v0.0.0-20190919134610-bf292b21730f
```